src/components/SidePanels/PanelBase.tsx

```diff
@@ -102,11 +103,19 @@ export const PanelBase: FC<PanelBaseProps> = ({
   }, [onVisibilityChange, dragLocked]);
 
   const style = useMemo(() => {
-    return visible ? {
+    const dynamicStyle = visible ? {
       height: detached ? height ?? '100%' : '100%',
       width: expanded ? "100%" : width ?? 320,
-    } : {};
-  }, [width, height, visible, detached, expanded]);
+    } : {
+      width: detached ? width ?? 320 : "100%",
+      height: detached ? 26 : undefined, // header height + 1px margin,
```

Comment:
is there anyway we can define 320 and 26 as constant variables - as these values seem arbitrary